src/api/ens.js

```diff
@@ -6,6 +6,12 @@ import { abi as oldResolverContract } from '@ensdomains/ens-022/build/contracts/
 
 import { abi as fifsRegistrarContract } from '@ensdomains/ens/build/contracts/FIFSRegistrar.json'
 
+oldResolverContract.map( (old, i) =>{
```

Comment:
Use `forEach` if you are not returning the array. 

Also do we want to do this programmatically everytime at runtime? Seems like something we could probably do before hand and just add the compiled and merged JSON

Reply:
One way is to create a task (eg: `yarn mergeResolvers` to merge, save it somewhere and the commit the file but I am hesitant to do that as I am trying to move away from generating any ABI files within `ens-app`. The alternative is to simply copy the old Resolver version  into `ensdomains/resolvers` then runs the task at `prepublishOnly`  so that it gets generated every time someone publish new resolver npm module. I suggest that I do that as a separate PR after this PR gets merged.

Reply:
Why not just have a single contract interface  that you use, which has all the methods you might need?

Reply:
Like https://github.com/ensdomains/resolvers/pull/5 ?